app/src/main/java/com/example/a2d_game_test/gameEngine/Game.java

```diff
@@ -94,29 +98,33 @@ public void draw(Canvas canvas) {
     }
 
     public void drawUpdatesPerSecond(Canvas canvas) {
-        String averageUpdatesPerSecond = Double.toString(this.gameLoop.getAverageUpdatesPerSecond());
-
-        int color = ContextCompat.getColor(getContext(), R.color.silver);
+        double averageUpdatesPerSecond = this.gameLoop.getAverageUpdatesPerSecond();
+        String textToDraw = this.getTextFormattedTODraw(UPDATES_PER_SECOND, averageUpdatesPerSecond);
 
+        int color = ContextCompat.getColor(getContext(), R.color.teal_200);
         Paint paint = new Paint();
+
         paint.setColor(color);
         paint.setTextSize(TEXT_SIZE);
 
-        // Random numbers
-        canvas.drawText("UpdatesPerSecond: " + averageUpdatesPerSecond, 156, 246, paint);
+        canvas.drawText(textToDraw, 0, TEXT_SIZE, paint);
```

Comment:
don't use text size for location

Reply:
why?

Reply:
because the name is not represent what its using for, so its confusing